src/material/core/_core.scss

```diff
@@ -4,9 +4,15 @@
 
 // Core styles that can be used to apply material design treatments to any element.
 @use './ripple/ripple';
-@use './focus-indicators/focus-indicators';
+@use 'style/focus-indicators';
```

Comment:
nit: should start with `./` so it obvious that it's a relative path and not an absolute one to an npm depepdency.